Packs/LINENotify/Integrations/LINENotify/README.md

```diff
@@ -0,0 +1,43 @@
+This is the Hello World integration for getting started.
+https://notify-bot.line.me/doc/en/
+This integration was integrated and tested with version xx of LINENotify
+
+## Configure LINENotify on Cortex XSOAR
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for LINENotify.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Required** |
+    | --- | --- |
+    | Token of LINE Group | True |
+
+4. Click **Test** to validate the URLs, token, and connection.
+## Commands
+You can execute these commands from the Cortex XSOAR CLI, as part of an automation, or in a playbook.
+After you successfully execute a command, a DBot message appears in the War Room with the command details.
+### LINE-send-message
+***
+Send message/notification to LINE Group
+
+
+#### Base Command
+
+`LINE-send-message`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| msg | Message to be sent. | Required | 
+
+
+#### Context Output
+
+There is no context output for this command.
+
+#### Command Example
```

Comment:
highly recommended to add an example for the command.
you can follow this documentation https://xsoar.pan.dev/docs/documentation/readme_file#run-demisto-sdk-generate-docs to generate the command example.

Reply:
Example:
!LINE-send-message msg="Send Message to Nicholas Group"
Result: 
![image](https://user-images.githubusercontent.com/49844292/126330894-d0ea47e5-74e1-484f-80f6-35503f070848.png)